packages/amazonq/.changes/next-release/Feature-8b5d9fca-a165-4b63-a880-3e8ea8ec0bd8.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Q feature dev to extract more relevant data when initializing"
```

Comment:
I wonder if we could make this more clear? Something like
Q feature dev: Only use relevant code + related files?

Reply:
Hey Josh, sounds good, I will push another revision